content/Setting Up Firebase on Android- Amaury Ricardo.md

```diff
@@ -0,0 +1,113 @@
+
+## Setting Up Firebase on Android.
+### What's Firebase?
+
+Firebase is a mobile platform that helps you quickly develop high-quality apps, grow your user base, and more. Firebase is made up of complementary features that you can mix-and-match to fit your needs, helping you to build apps fast, without managing infrastructure. Firebase gives you functionality like analytics, databases, messaging, and crash reporting so you can move quickly and focus on your users. [[from google]](https://developer.android.com/studio/write/firebase)
+
+Was created by Envolved between 2011-2014, with just a couple o features (Realtime db, Hosting and Authentication) and was acquired by Google in 2014. A year later, in 2015, Google acquired Divshot, an HTML5 web-hosting platform, to merge it with Firebase, and integrated a lot of google services as well, in 2016 acquired LaunchKit a toolkit for mobile app development and in 2017 Fabric as analytics tool. Every year Google release at least one new functionality for Firebase in order to improve the mobile and web development.
+
+### What can we do with Firebase?
+
+Firebase provides a lot of tools that we will talk about in next posts but resuming Firebase allow as to:
+- Push notifications
+- Analytics
+- A/B Testing
+- Authentication
+- Real time db
+- App distributions
+- Google cloud: Hosting, Storage
+- Machine Learning
+
+and a lot of others tools that I invited you to test.
+
+### Setting up on Android
+
+1. **Adding dependencies**
+
+On your project  Gradle file "build.gradle"
+
+    ```java
+    repositories {
+        google()
+        jcenter()
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
+    }
+
+    dependencies {
+    		....        
+    		classpath 'com.google.gms:google-services:4.3.2'  // Google Services plugin
+        classpath 'io.fabric.tools:gradle:1.31.2'  // Crashlytics plugin
+    		....
+    }
+
+    ```
+
+On your module (app) Gradle file "build.gradle"
+
+    ```java
+    apply plugin: 'com.android.application'
+
+    android {
+       ....
+    }
+    dependencies {
+    		.....
+    	// Add the Firebase SDK for Google Analytics
+      implementation 'com.google.firebase:firebase-analytics:17.2.1'
+    }
+
+    // Add the following line to the bottom of the file
+    apply plugin: 'com.google.gms.google-services'
+    ```
+
+2. **Create the project on your Firebase console**
+
+    Open your [firebase console](https://console.firebase.google.com/)  and add a new project:
+
+    ![Add firebase](images/fb1.png)
+
+    Once you create the project in order to add a new Android app:
```

Comment:
`once ... in order to .. doesn't make much sense. ``Once you create the project in Firebase console, add a new Android app.` or something like that.